src/java/org/apache/cassandra/repair/CassandraRepairJob.java

```diff
@@ -86,13 +86,10 @@ public class RepairJob extends AsyncFuture<RepairResult> implements Runnable
      *  @param session RepairSession that this RepairJob belongs
      * @param columnFamily name of the ColumnFamily to repair
      */
-    public RepairJob(RepairSession session, String columnFamily)
+    public CassandraRepairJob(RepairSession session, String columnFamily)
     {
-        this.session = session;
-        this.taskExecutor = session.taskExecutor;
+        super(session, columnFamily);
```

Comment:
That's not 100% straightforward because a table can be running both `Accord` and `Paxos` and can migrate back and forth in either direction. The property your describing maybe maps better to whether the range being repaired is currently running on Accord or Paxos?

Reply:
I left this alone until we decide how standard repair is going to interoperate with Accord.